core/src/main/java/org/apache/calcite/util/Sarg.java

```diff
@@ -111,7 +117,7 @@ public StringBuilder printTo(StringBuilder sb,
       RangeSets.forEach(r, printer);
     });
     if (containsNull) {
-      sb.append(", null");
+      sb.append(" OR NULL");
```

Comment:
Is there a reason for the change?
It looks like the only place where OR is used in the string representation of Sarg. Why is OR needed here?

I would prefer to have a consistent delimiter (e.g. comma always) rather than different delimeters in different places.

Reply:
Null is not like other values. The new formatting scheme is more intuitive, I think.

Reply:
How about `, or NULL`?

Reply:
Another thought: what if `null` is placed in front of other predicates? Then it would be easier to recognize, especially in case there are lots of ranges.